### DISCUSSION TITLE
ColorField not applying value on paste (only applies on blur/click away) - Is this known, intended or is there any more info?

### DISCUSSION THREAD

**Discussion by rlEsteban** [adobe/react-spectrum#7206] [Q&A]
The ColorField is not applying a value on paste and only applying on blur so I'm having to manually cover the case for a UX issue in my work. I'm wondering if this is something that's known and intentionally not handled, if there's plans to cover this case or if there's any further info about this? Thanks in advance for the information!

Examples here:
https://react-spectrum.adobe.com/react-aria/useColorField.html#controlled
https://react-spectrum.adobe.com/react-aria/ColorField.html#controlled

I just did a quick search through the discussions to see if I could find anything related but didn't see anything.

**Comment by snowystinger** [comment]
Thanks for the discussion. This is very similar to https://github.com/adobe/react-spectrum/issues/1879
I think colorfield has the same consideration. It is confusing if the color converts to something that isn't the text you were pasting.

Also related https://github.com/adobe/react-spectrum/pull/6520
If the value formats differently on paste, screen reader users may be confused until we can solve this PR
